src/kirby/components/segmented-control/segmented-control.component.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import * as ionic from '@ionic/angular';
+
+import { MockComponent } from 'ng-mocks';
+
+import { SegmentedControlComponent } from './segmented-control.component';
+import { SegmentItem } from './segment-item';
+import { BadgeComponent } from '../badge/badge.component';
+
+describe('SegmentedControlComponent', () => {
```

Comment:
A unit test is missing for the `onSegmentClick`-method

Reply:
I've added an unfinished test - there is an issue with testing the ionic segment-button that I can't seem to get around. 
Will correct the problem on another branch, to get this one out of the way.